src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.spec.ts

```diff
@@ -39,140 +38,151 @@ describe('TableComponent', () => {
   beforeEach(() => {
     fixture = TestBed.createComponent(TableComponent);
     component = fixture.componentInstance;
-  });
 
-  beforeEach(() => {
-    component.data = createFakeData(100);
+    component.data = createFakeData(10);
```

Comment:
I'm not sure why you reduced the data to search through and as described above made each column unique to any other column through the new data structure and this limitation of data.